src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import ToDo from '../../types/ToDo';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  todo: ToDo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { title, completed, id, userId } = todo;
+
+  const foundedUser =
+    usersFromServer.find(checkedUser => checkedUser.id === userId) || '';
```

Comment:
[Prepare data in one place (App component) and pass it to child components] - You should not import 'usersFromServer' directly in the 'TodoInfo' component. Instead, prepare the data in the parent component (e.g., 'App') and pass it down via props.